src/augment/fields.js

```diff
@@ -186,6 +190,66 @@ export const toSnakeCase = name => {
     .join('');
 };
 
+/**
+ * Builds the AST definition for _updated node property
+ * adding an _updated field to graphql types
+ */
+export const buildNeo4jLastUpdatedTimestampField = ({
+  typeName,
+  propertyOutputFields,
+  nodeInputTypeMap,
+  config,
+  isRelationship = false
+}) => {
+  const queryTypeNameLower = OperationType.QUERY.toLowerCase();
+  if (
+    !shouldAugmentType(config, queryTypeNameLower, typeName) ||
+    isRelationship
+  ) {
+    return [propertyOutputFields, nodeInputTypeMap];
+  }
+
+  const neo4jUpdatedTimestampConfig = {
+    name: Neo4jSystemUpdatedTimestampField,
+    type: {
+      name: GraphQLFloat.name
+    }
+  };
+  const systemIDIndex = propertyOutputFields.findIndex(
+    e => e.name.value === Neo4jSystemUpdatedTimestampField
+  );
+
+  const systemIDField = buildField({
+    name: buildName({ name: neo4jUpdatedTimestampConfig.name }),
+    type: buildNamedType({
+      name: GraphQLFloat.name

```

Comment:
Why `GraphQLFloat` here? Is it because updated time is stored as a float? Also, should this var be called `systemIdField`, or something related to the updated timestamp? Same as `systemIDIndex` above

Reply:
GraphQL int type is not capable to keep values which overflow 32bits, ideally a long type is needed here but as GraphQL doesn't provide it I decided to use float (64bit)